packages/neos-ui-extensibility-webpack-adapter/README.md

```diff
@@ -0,0 +1,134 @@
+# @neos-project/neos-ui-extensibility-webpack-adapter
+
+> Zero¹ configuration, highly opinionated Webpack 4 + Babel plugin buildstack for the Neos CMS UI.
+
+## Installation
+```bash
+yarn add -D @neos-project/neos-ui-extensibility-webpack-adapter
+```
+
+## Alternative: Esbuild, vanilla Webpack5 (or any buildsystem with alias resolution)
+If you prefer a less opinionated and modern build stack - or better, just want to choose and fully configure the bundler yourself - visit [@neos-project/neos-ui-extensibility](https://github.com/neos/neos-ui/blob/8.3/packages/neos-ui-extensibility) to use the Neos UI plugin API directly.
+
+Note that this adapter uses a build stack created some time ago (webpack4 + babel) and exists mostly for convenience/backwards-compatibility, but is not state of the art and might never be again. So in case you're thinking about patching a new loader/option into the `webpack.config.js` you might as well just use webpack directly or try out esbuild ;).
```

Comment:
```suggestion
Note that this adapter uses a build stack created some time ago (Webpack4 + Babel) and exists mostly for convenience/backwards-compatibility, but is not state of the art and might never be again. So in case you're thinking about patching a new loader/option into the `webpack.config.js` you might as well just use Webpack directly or try out esbuild ;).
```